lefthook.yml

```diff
@@ -0,0 +1,14 @@
+pre-push:
+  commands:
+    coverage:
+      run: |
+        npm run build &&
+        npm run test:coverage
+
+pre-commit:
+  parallel: true
+  commands:
+    fmt:
+      run: npm run fmt:check
+    test:
+      run: npm run test:size
```

Comment:
I suggest adding all the tasks of ci check flow: https://github.com/google/zx/blob/f1ca807532cce3a9523826a5f6c884d60a85c593/.github/workflows/test.yml#L32

Reply:
d1